src/normalize/entities.ts

```diff
@@ -164,6 +174,25 @@ const commonMailItemFields = {
 	attach: ['attachments', AttachmentsInfo]
 };
 
+const SendMessageFields = new Entity({
```

Comment:
you should be able to get a lot of these keys from commonMailItemFields (which also uses commonMessageFields) instead of declaring them all over again

Reply:
in fact, you might be able to just modify commonMailItemFields to add anything extra that was missing

Reply:
Thanks for pointing out this @jkester1986.
I have fixed it now please review.